content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,69 @@
+# Machine Learning Model Registry
+
+A **model registry** is a central catalog for data artifacts produced in
+multiple data science projects. Different models can be discovered, audited,
+shared, tested, and deployed from here. DVC works on top of Git to enable these
+capabilities on your existing ecosystem. And other tools from
+[Iterative](https://iterative.ai/) provide even better features and usability.
```

Comment:
better -> more?

Reply:
they are not better ... they are additional

Reply:
Changed to more features and better usability